.travis.yml

```diff
@@ -17,5 +17,6 @@ os:
 script:
     - export PARSL_TESTING="true"
     - pip install -r test-requirements.txt
-    - nosetests -v parsl/tests/test_threads parsl/tests/test_data parsl/tests/test_checkpointing
+    - nosetests -v parsl/tests/test_threads parsl/tests/test_data
+    - for test in parsl/tests/test_checkpointing/test*; do nosetests -v $test; done
```

Comment:
Why is this necessary? Nose will traverse through directory levels. Also, maybe we should change `-v` -> `-vx` so things fail faster.

Reply:
Agreed on -v -> -vx.
Nosetests is doing something unusual when running these tests from a directory. They pass when called one by one. This was easier that debugging what nose is doing.